adapters/rubicon/rubicontest/exemplary/simple-video.json

```diff
@@ -120,8 +120,6 @@
           "w": 1024,
           "h": 576
         },
-        "bidfloor": 1,
-        "bidfloorcur": "EuR",
```

Comment:
This newly coded `bidFloor` currency conversion functionality seems to not be tested in any JSON test. Can we include one? I realize a test case asserting this functionality has already been coded but I believe a JSON test would be useful to prove end to end functionality even further.

Reply:
Gus, guys, please correct me if I'm wrong. 
I think test cases with "EUR" were removed due to this update:
```
if bidFloor > 0 {
-		if strings.ToUpper(bidFloorCur) == "EUR" {
-			return bidFloor * 1.2, "USD"
+		if strings.ToUpper(bidFloorCur) != "USD" {
+			return reqInfo.ConvertCurrency(bidFloor, bidFloorCur, "USD")
		}

```
EUR is no longer hardcoded and it uses real currency conversion in `GetRates` function. To make it work in test mock function is used:
```
setMock:          func(m *mock.Mock) { m.On("GetRate", "EUR", "USD").Return(1.2, nil) },
```
Not sure if it's possible to control it in json tests.

BidFloor functionality is covered in `TestOpenRTBRequestWithDifferentBidFloorAttributes`

Reply:
You have described all my thoughts, thanks @VeronikaSolovei9

Reply:
Correct. Thank you @VeronikaSolovei9